sysutils/z/Portfile

```diff
@@ -3,7 +3,8 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        rupa z 1.11 v
+name                z
+homepage            https://github.com/rupa/z
```

Comment:
Were you not able to remove these essentially redundant lines?

Reply:
actually i added both of them since it looks better to me to set them explicit.

i am going to remove the **homepage** line.

while as far as i understand the **name** line is needed for the check at line 15:
     `if {${name} eq ${subport}} {`
i guess it become not redundant because of the existence of the subport.

am i missing something?

Reply:
`github.setup` sets `homepage` for you.